packages/features/src/Domain/Feature/features.json

```diff
@@ -248,7 +248,6 @@
     "url": "#{url_prefix}/components/mfa-link",
     "marketing_url": "",
     "download_url": "https://github.com/standardnotes/mfa-link/archive/1.2.4.zip",
-    "content_type": "SN|Component",
```

Comment:
But do we still want to handle it as a component now that it will be natively integrated on the preferences pane?

Reply:
If this PR + native 2FA are being shipped at the same time, then this entire feature payload can be deleted. Otherwise if there is a legacy grace period, this will be a component.

Reply:
Makes sense, will keep it as component for now